cmd/removeEnumerator.go

```diff
@@ -52,17 +53,17 @@ func newRemoveEnumerator(cca *cookedCopyCmdArgs) (enumerator *copyEnumerator, er
 		copyHandlerUtil{}.appendQueryParamToUrl(rawURL, cca.sourceSAS)
 	}
 
-	sourceTraverser, err = initResourceTraverser(rawURL.String(), cca.fromTo.From(), &ctx, &cca.credentialInfo, nil, &cca.listOfFilesLocation, cca.recursive, func() {})
+	sourceTraverser, err = initResourceTraverser(rawURL.String(), cca.fromTo.From(), &ctx, &cca.credentialInfo, nil, cca.listOfFilesChannel, cca.recursive, func() {})
 
 	// report failure to create traverser
 	if err != nil {
 		return nil, err
 	}
 
 	transferScheduler := newRemoveTransferProcessor(cca, NumOfFilesPerDispatchJobPart)
-	includeFilters := buildIncludeFilters(cca.includePatterns, false) // TODO: Add include-path in here?
+	includeFilters := buildIncludeFilters(cca.includePatterns) // TODO: Add include-path in here?
 	excludeFilters := buildExcludeFilters(cca.excludePatterns, false)
-	includePathFilters := buildIncludeFilters(cca.includePathPatterns, true)
+	includePathFilters := buildIncludeFilters(cca.includePathPatterns)
```

Comment:
Why is this line still here?  I was expecting it to go, with the change to use listOfFilesChannel

Reply:
Good point, there was no need for this.